addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -65,6 +65,11 @@ export class PaymentScreen extends Component {
         if (this.payment_methods_from_config.length == 1 && this.paymentLines.length == 0) {
             this.addNewPaymentLine(this.payment_methods_from_config[0]);
         }
+
+        //Activate the invoice option for refund orders if the original order was invoiced.
+        if (this.currentOrder.refunded_order_id?.is_to_invoice()) {
```

Comment:
currently `refunded_order_id` is got undefined! please check once if issue still occurs

Reply:
Could you clarify in which scenario? 
Based on the current condition, the refunded_order_id should only have a value for orders that are indeed refunds.